pom.xml

```diff
@@ -96,6 +105,7 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
             <version>1.7.6</version>
+            <scope>provided</scope>
```

Comment:
Do not want to cause conflict with a runtime that already has slf4j. Multiple instances and versions are a bit of a problem area since slf4j uses static binding.  

Sent from my iPhone

> On 18-Apr-2016, at 4:44 PM, Santanu <notifications@github.com> wrote:
> 
> In pom.xml:
> 
> > @@ -96,6 +105,7 @@
> >              <groupId>org.slf4j</groupId>
> >              <artifactId>slf4j-api</artifactId>
> >              <version>1.7.6</version>
> > +            <scope>provided</scope>
> why is slf4j-api provided? the log4j etc bindings might be provided.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>